Tools/sitl_run.sh

```diff
@@ -100,6 +100,12 @@ done
 
 export PX4_SIM_MODEL=${model}
 
+if [[ "$sitl_bin" == *"px4_sitl_sih"* ]]; then
```

Comment:
I'm comaring the string command with wildcard to check if it contains px4_sitl_sih, is there a cleaner way to do that?